index.html

```diff
@@ -1004,6 +1006,103 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">protocol_handlers</dfn></code> member is an <a>array</a>
```

Comment:
Don't export until someone asks for it. 

```suggestion
          The [=manifest's=] <code><dfn  data-dfn-for="manifest">protocol_handlers</dfn></code> member is an <a>array</a>
```